genesis/genesis.go

```diff
@@ -83,18 +83,24 @@ func NewFile(isQuorum bool, options ...Option) string {
 
 func Save(dataDir string, genesis *core.Genesis, isQuorum bool) error {
 	filePath := filepath.Join(dataDir, FileName)
+	err, jsonStr := ToString(genesis, isQuorum)
+	if err != nil {
+		return err
+	}
+	return ioutil.WriteFile(filePath, []byte(jsonStr), 0600)
+}
 
+func ToString(genesis *core.Genesis, isQuorum bool) (error, string) {
```

Comment:
The return values should be `(string, error)`.

Reply:
OK